appsec/src/extension/ddtrace.c

```diff
@@ -123,6 +147,7 @@ void dd_trace_startup()
 
     if (get_global_DD_APPSEC_TESTING()) {
         _register_testing_objects();
+        _setup_testing_telemetry_functions();
```

Comment:
I coudln't get why this is within `APPSEC_TESTING`. I'm probably missing something

Reply:
Ok. Makes sense